Helper/Order.php

```diff
@@ -2379,6 +2389,19 @@ public function updateOrderPayment($order, $transaction = null, $reference = nul
 
         $payment_transaction->save();
 
+        if ($transactionType == Transaction::TYPE_VOID || ($transactionType == Transaction::TYPE_CAPTURE && $this->isFullyCaptured($order, $transaction))) {
+            $orderTransaction = $this->transactionRepository->getByTransactionType(
```

Comment:
Do I understand correctly that `getByTransactionType` never throws an exception?

Reply:
It does. InputException
I was going to add it before merging.